C++/081_Search in Rotated Sorted Array II.cpp

```diff
@@ -0,0 +1,79 @@
+//   Problem 81, Search in Rotated Sorted Array II
```

Comment:
could you use multi-line comment like @forest5013 for problem description? Let's all keep the same format. Thanks!